New version: PlutoDependencyExplorer v1.0.3

JuliaRegistrator opened the pull request:
- Registering package: PlutoDependencyExplorer
- Repository: https://github.com/JuliaPluto/PlutoDependencyExplorer.jl
- Created by: @fonsp
- Version: v1.0.3
- Commit: 43102e988108a416c5d9a2934832d27a1a57ce2a
- Reviewed by: @fonsp
- Reference: https://github.com/JuliaPluto/PlutoDependencyExplorer.jl/commit/43102e988108a416c5d9a2934832d27a1a57ce2a#commitcomment-138375876
- Description: Pluto's dependency sorting algorithm
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd69d3cd5a453c0575b16691063394ebdf0400eab74588508d5141661b0207f614163d1bca984287ee26703216d508d665f2f87b1afed53512b456d7642e278eec4c94625e2fa9e5209ff5343cfc0ff2eb9262abd427b9687b08b65992489b0ce085654771ef317ce7e6c9564f6d11bca22100ceb9e586ad73512719ff16e9d8c92b3a532bbf3191f23ecc95b843159055892bd629d8329bac7e6764ea2d68712539ea224170d319693e3308e0c1b0451d4230b218df8a7bf6982d0f7a11093029c -->

fonsp commented:
hold on

fonsp commented:
hold on

> Op 7 feb 2024, om 16:22 heeft github-actions[bot] ***@***.***> het volgende geschreven:
> 
> 
> Your new version pull request met all of the guidelines for auto-merging and is scheduled to be merged in the next round.
> 
> If you want to prevent this pull request from being auto-merged, simply leave a comment. If you want to post a comment without blocking auto-merging, you must include the text [noblock] in your comment. You can edit blocking comments, adding [noblock] to them in order to unblock auto-merging.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/JuliaRegistries/General/pull/100410#issuecomment-1932275337>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABU4YBSZ4LMRTOLL4X3HBM3YSOL2FAVCNFSM6AAAAABC6AR3SKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZSGI3TKMZTG4>.
> You are receiving this because you were mentioned.
>

JuliaRegistrator committed:
New version: PlutoDependencyExplorer v1.0.3

UUID: 72656b73-756c-7461-726b-72656b6b696b
Repo: https://github.com/JuliaPluto/PlutoDependencyExplorer.jl.git
Tree: dd77d591d2f72c49541dfd245630b3535b4ff7c4

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
